src/components/enterprise-user-subsidy/enterprise-offers/data/hooks.js

```diff
@@ -77,22 +76,12 @@ export const useEnterpriseOffers = ({
       return;
     }
 
-    const enterpriseHasActiveCoupons = enterpriseCoupons.length > 0;
-    const enterpriseHasActiveSubscription = !!(customerAgreementConfig?.subscriptions ?? []).find(({
-      startDate,
-      expirationDate,
-    }) => hasValidStartExpirationDates({
-      startDate,
-      expirationDate,
-    }));
-
     // For MVP we will only support enterprises with one active offer
-    const enterpriseHasOneOffer = enterpriseOffers.length === 1;
+    const enterpriseHasOneOffer = (enterpriseOffers.length < 2 && enterpriseOffers.length > 0);
```

Comment:
I would like to figure out this mystery before we merge the PR as I would think these statements are functionally equivalent.

I ran this on firefox 105.0.1 (64-bit):
```
enterpriseOffers = [Object]
Array [ Object() ]

enterpriseOffers.length
1
enterpriseOffers.length === 1
true
enterpriseOffers.length < 2 && enterpriseOffers.length > 0
true
```